app/components/img-selector.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    thumbnail: null,
+    actions: {
+        uploadImage: function(e) {
+            var self = this;
+
+            //var canvas = this.$().find('.img-selector-canvas')[0];
+            //var ctx = canvas.getContext('2d');
+
+            var reader = new window.FileReader();
+            reader.onload = function(event){
+                /*
+                var img = new Image();
```

Comment:
There are several sections of commented-out code in this file; could you please clean them up? (low priority, but nice)